lib/registrant.rb

```diff
@@ -0,0 +1,23 @@
+class Registrant
+  attr_reader :name,
+              :age,
+              :license_data,
+              :earn_permit
+
+  def initialize(name, age, permit  = false)
+    @name = name
+    @age = age
+    @permit = permit
+    @license_data = {:written=>false, 
+                     :license=>false, 
+                     :renewed=>false}
+  end
+
+  def permit?
+    @permit == true 
```

Comment:
Here you would just call `@permit`, which is set in your initialize as `false`.  The `earn_permit` method will end up updating this in your tests.